test/runtests.jl

```diff
@@ -12,6 +12,14 @@ else
     throwing_function(bt) = bt[1]
 end
 
+const rseed = Ref(Base.GLOBAL_RNG)  # to get new random directories (see #24445)
+function randtmp()
+    srand(rseed[])
+    dirname = joinpath(tempdir(), randstring(10))
+    rseed[] = Base.GLOBAL_RNG
```

Comment:
I don't understand the point of this line: isn't rseed[] already equal to Base.GLOBAL_RNG?

Reply:
I can't remember anymore.